### PR TITLE
MEPTS 1234 | Unit Tests For TX_PVLS Breastfeeding Cohort

### PR DESCRIPTION
**Pull request by Emmanuel0077**

### CODE REVIEW

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

File: api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/TxPvlsCohortQueriesTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    context.addParameterValue("endDate", endDate);
+    context.addParameterValue("location", location);
+  }
+
+  @Test
+  public void getWomanWhoAreBreastfeedingShouldNOTpass() throws EvaluationException {
+    CohortDefinition cd = txPvlsCohortQueries.getBreastfeedingPatients();
+
+    HashMap<Parameter, Object> parameters = new HashMap<>();
+    parameters.put(new Parameter("startDate", "Start Date", Date.class), getStartDate());
```
Comment (eliazmuchave) [inline api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/TxPvlsCohortQueriesTest.java]: This is never used

File: api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/TxPvlsCohortQueriesTest.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    assertEquals(2, evaluatedCohort.getMemberIds().size());
+    assertFalse(evaluatedCohort.getMemberIds().contains(1002));
+  }
+
+  @Test
+  public void getNumberOfPatientshouldPass() throws EvaluationException {
+    CohortDefinition cd = txPvlsCohortQueries.getBreastfeedingPatients();
+
+    HashMap<Parameter, Object> parameters = new HashMap<>();
+    parameters.put(new Parameter("startDate", "Start Date", Date.class), getStartDate());
```
Comment (eliazmuchave) [inline api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/TxPvlsCohortQueriesTest.java]: This is never used

**Review by eliazmuchave — CHANGES_REQUESTED** [review]

File: api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/TxPvlsCohortQueriesTest.java (hunk 2 above)
Comment (Emmanuel0077) [reply api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/library/cohorts/TxPvlsCohortQueriesTest.java]: I will remove it, as soon as possible

**Pull request merged**